test/assert.js

```diff
@@ -877,6 +903,22 @@ describe('assert', function () {
       errSet.add({1: 20});
       errSet.add('number');
 
+      // Tests for the deep variations of the keys assertion
```

Comment:
I'm thinking a test should be added to the non-deep variety of assertion above demonstrating that it uses strict equality instead of deep. For example:

```js
assert.doesNotHaveAnyKeys(testSet, {thisIs: 'anExampleObject'});
```